src/server/pegasus_write_service_impl.h

```diff
@@ -349,13 +350,13 @@ class pegasus_write_service::impl : public dsn::replication::replica_base
                 decree, dsn::string_view(), dsn::string_view(), 0);
         }
         if (resp.error) {
-            _rocksdb_wrapper->clear_up_write_batch();
+            rocksdb_write_batch_cleaner cleaner(_rocksdb_wrapper.get());
```

Comment:
Define it in the function body scope, then you can only define it once.

Reply:
OK, I got what you mean.